### PR TITLE
Improve the styles for igniteui docFX template

### LINKED ISSUE #318: Improve side-nav design of the documentation
Improve the side-nav design of the tree view navigation with topic titles on the left for the template of Ignite UI for Angular.

### PR DESCRIPTION
**Pull request by desig9stein**
Closes #318

### CODE REVIEW

**Comment by StefanIvanov** [comment]
I have two requests @desig9stein :

- Upon selecting strikethrough text the style for strikethrough removes the line, it should still be there with the selected style applied.
- The details element should have the same style when expanded/collapsed use the blue color for the text for both

**Comment by desig9stein** [comment]
> * Upon selecting strikethrough text the style for strikethrough removes the line, it should still be there with the selected style applied.

@StefanIvanov This can't be fixed, the implementation of the CSS pseudo selector `::selection` works that way, it removes all `text-decoration` styles from any text.

Here is a link to the [StackOverflow bug report](https://stackoverflow.com/questions/20657855/css3-selection-selector-not-working-properly-on-anchor-tags)

**Comment by StefanIvanov** [comment]
I think we are good to merge once @zdrawku approves this

**Review by zdrawku — APPROVED** [review]

**Pull request merged**
